### PR TITLE
chore(h2): add section about removal of camunda-h2-webapp

### PR DESCRIPTION
**Pull request by mboskamp**
Related to CAM-14482

### CODE REVIEW

**Comment by mboskamp** [comment]
Hi @christinaausley, could you have a look at the documentation section I added before I port it to all the branches? :)

**Review by christinaausley — APPROVED** [review]
The only change in my latest commit is largely just writing "web app" rather than "webapp" or "web application" -- looks good.

**Comment by mboskamp** [comment]
Thank you, @christinaausley! 🎉

File: content/update/minor/717-to-718/_index.md [hunk 1]
```diff
@@ -177,4 +179,18 @@ to `#reconfigure(List, ExternalTaskClientException)`.
 Before 7.18, the exception cause was wrapped in a `FetchAndLockException` and passed to `#reconfigure(List, Exception)`.
 With this version, we streamlined the exception types thrown when calling operations on the `ExternalTaskService` with 
 the exceptions passed to `#reconfigure(List, ExternalTaskClientException)` in case a "fetch and lock" request fails.
-This is why we removed the `FetchAndLockException` and replaced it with an `ExternalTaskClientException`.
\ No newline at end of file
+This is why we removed the `FetchAndLockException` and replaced it with an `ExternalTaskClientException`.
+
+# Discontinue Camunda H2 console web app
```
Comment (koevskinikola) [inline content/update/minor/717-to-718/_index.md:184]: @mboskamp do you think it makes sense to mention the discontinuation in the Enterprise Announcements page? We've done something similar in the past with the JavaScript External Task client: https://docs.camunda.org/enterprise/announcement/#camunda-platform-7-13

**Review by koevskinikola — APPROVED** [review]
Looks good, my comments is for us to consider if we should document this in the `camunda-docs-static` repo as well.

File: content/update/minor/717-to-718/_index.md (hunk 1 above)
Comment (mboskamp) [reply content/update/minor/717-to-718/_index.md:184]: @koevskinikola, I don't think the discontinuation of the h2 console webapp should be mentioned on the enterprise announcement page. This page only covers changes to our supported environments. We only discontinue support for one feature of H2 and don't change anything in terms of support for h2 in general.

**Pull request merged**